ui/docker/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:lts-jessie as build-stage
+
+ARG ENABLE_LOCAL_LOGIN=YES
+ENV REACT_APP_ENABLE_LOCAL_LOGIN=$ENABLE_LOCAL_LOGIN
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+ENV PATH /usr/src/app/node_modules/.bin:$PATH
+COPY package.json yarn.lock /usr/src/app/
+RUN yarn install
+COPY . /usr/src/app
+RUN yarn run build
+
+FROM nginx:1.15.9-alpine
+
+ARG NGINX_CONFIGURATION_FILE=local.conf
```

Comment:
```suggestion
```
It would be better to build the image with the prod config and use a volume in compose to overwrite it.